extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/destination/MSQSelectDestination.java

```diff
@@ -30,6 +30,10 @@ public enum MSQSelectDestination
    * Writes all the results directly to the report.
    */
   TASKREPORT("taskReport", false),
+  /**
+   * Writes the results as rows to a location.
```

Comment:
location is generic, can expand a bit further.

Reply:
Updated the comment